.teamcity/settings.kts

```diff
@@ -1079,112 +1079,140 @@ object WpDesktop_DesktopE2ETests : BuildType({
-                set -o nounset
-                set -o pipefail
-
-                # Update composer
-                composer install
-
-                # Install modules
-                yarn install
-            """.trimIndent()
-            dockerImagePlatform = ScriptBuildStep.ImagePlatform.Linux
-            dockerPull = true
-            dockerImage = "%docker_image_wpcom%"
-            dockerRunParameters = "-u %env.UID%"
-        }
-        script {
-            name = "Run tests"
-            executionMode = BuildStep.ExecutionMode.RUN_ON_FAILURE
-            scriptContent = """
-                #!/bin/bash
-
-                # Update node
-                . "${'$'}NVM_DIR/nvm.sh" --no-use
-                nvm install
-
-                set -o errexit
-                set -o nounset
-                set -o pipefail
-
-                export JEST_JUNIT_OUTPUT_NAME="results.xml"
-                export JEST_JUNIT_OUTPUT_DIR="../../test_results/editing-toolkit"
-
-                cd apps/editing-toolkit
-                yarn test:js --reporters=default --reporters=jest-junit --maxWorkers=${'$'}JEST_MAX_WORKERS
-            """.trimIndent()
-            dockerImagePlatform = ScriptBuildStep.ImagePlatform.Linux
-            dockerPull = true
-            dockerImage = "%docker_image_wpcom%"
-            dockerRunParameters = "-u %env.UID%"
-        }
-        script {
-            name = "Build artifacts"
-            executionMode = BuildStep.ExecutionMode.RUN_ON_FAILURE
-            scriptContent = """
-                #!/bin/bash
-
-                # Update node
-                . "${'$'}NVM_DIR/nvm.sh" --no-use
-                nvm install
-
-                set -o errexit
-                set -o nounset
-                set -o pipefail
-
-                export NODE_ENV="production"
-
-                cd apps/editing-toolkit
-
-                # Update plugin version in the plugin file.
-                sed -i -e "/^\s\* Version:/c\ * Version: %build.number%" -e "/^define( 'PLUGIN_VERSION'/c\define( 'PLUGIN_VERSION', '%build.number%' );" ./editing-toolkit-plugin/full-site-editing-plugin.php
-
-                # Update plugin stable tag in readme.txt.
-                sed -i -e "/^Stable tag:\s/c\Stable tag: %build.number%" ./editing-toolkit-plugin/readme.txt
-
-                yarn build
-
-                cd editing-toolkit-plugin/
-                zip -r ../../../editing-toolkit.zip .
-            """.trimIndent()
-            dockerImagePlatform = ScriptBuildStep.ImagePlatform.Linux
-            dockerPull = true
-            dockerImage = "%docker_image_wpcom%"
-            dockerRunParameters = "-u %env.UID%"
-        }
-    }
+	name = "Editor ToolKit"
+
+	artifactRules = "editing-toolkit.zip"
+
+	buildNumberPattern = "%build.prefix%.%build.counter%"
+
+	params {
+		param("build.prefix", "3")
+	}
+
+	vcs {
+		root(WpCalypso)
+		cleanCheckout = true
+	}
+
+	triggers {
+		vcs {
+			triggerRules = "+:apps/editing-toolkit/**"
```

Comment:
I'd suggest adding
```
branchFilter = """
	+:*
	-:pull*
""".trimIndent()
```

To make sure it doesn't get automatically triggered for external PRs (the `pullRequest` feature will expose branches named `pull/xxx` for every Pull Request from an external fork)